nix/release.nix

```diff
@@ -5,7 +5,9 @@ let
   tex = import ./tex-toolchain.nix { inherit pkgs; };
   inherit (import sources."gitignore.nix" { inherit (pkgs) lib; }) gitignoreSource;
 in
-pkgs.callPackage ./build.nix {
-  inherit gitignoreSource;
-  inherit tex;
+{
+  pdf = pkgs.callPackage ./build.nix {
```

Comment:
Why did you made this an attribute set again? I removed it as I thought it's not relevant.

Reply:
It's convention and you never now whether someone wants more attributes, e.g. HTML output or something.

Reply:
> It's convention 

Can this be found in the Nix manual?